logstream/logstream_user_loki_test.go

```diff
@@ -53,12 +53,33 @@ func TestExampleLokiStreaming(t *testing.T) {
 				LogProducerTimeout:    &blockchain.StrDuration{Duration: 10 * time.Second},
 				LogProducerRetryLimit: ptr.Ptr(uint(10)),
 			}
+			loggingConfig.Loki = &config.LokiConfig{
+				TenantId: ptr.Ptr("CHANGE-ME"),
+				Endpoint: ptr.Ptr("CHANGE-ME"),
+			}
+			loggingConfig.Grafana = &config.GrafanaConfig{
+				BaseUrl:      ptr.Ptr("CHANGE-ME"),
+				DashboardUrl: ptr.Ptr("CHANGE-ME"),
+				BearerToken:  ptr.Ptr("CHANGE-ME"),
+			}
 
 			lw, err := logstream.NewLogStream(t, &loggingConfig)
 			require.NoError(t, err)
-			err = d.ConnectLogs(lw)
-			require.NoError(t, err)
+			for _, c := range d.containers {
+				err = lw.ConnectContainer(ctx, c, "")
+				require.NoError(t, err)
+			}
 			time.Sleep(5 * time.Second)
```

Comment:
Just seeing this now, why the hard sleep? Can we make it dynamic?

Reply:
Ah, this sleep is here so that the test container we have started earlier have time to emit some logs before test ends :-)